projects/packages/forms/src/contact-form/class-feedback.php

```diff
@@ -1221,4 +1222,42 @@ private function get_computed_consent( $post_data, $form ) {
 
 		return false;
 	}
+
+	/**
+	 * Get a field by its original form ID.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @param string $id Original form field ID.
+	 * @return Feedback_Field|null
+	 */
+	public function get_field_by_form_field_id( $id ) {
+		if ( ! is_string( $id ) || $id === '' ) {
+			return null;
+		}
+		foreach ( $this->fields as $field ) {
+			$form_field_id = $field->get_form_field_id();
+			if ( is_string( $form_field_id ) && $form_field_id !== '' && $form_field_id === $id ) {
```

Comment:
Again, given the null vs string discussion above, this comparison should be simpler, you wouldn't need to check if `is_string`, you'd only check for equality :)

Reply:
Also updated!